### ISSUE TITLE
crowdloan lease period end date of parachain not showing

### ISSUE THREAD

**Issue by muddlebee** [polkadot-js/apps#9919]
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot-rpc.dwellir.com#/parachains/crowdloan

showing only for Invarch below

![image](https://github.com/polkadot-js/apps/assets/8139783/29c6da03-5a41-4147-b5d5-bbef8e5d3c92)

**Comment by paullallier** [comment]
I don't think this is a bug - I think the others weren't showing and end date because the end date was already in the past?

**Comment by JelliedOwl** [comment]
I don't think this is a bug - I think the others weren't showing an end date because the end date was already in the past?

**Comment by ap211unitech** [comment]
> I don't think this is a bug - I think the others weren't showing an end date because the end date was already in the past?

that's right. This is how it's being calculated - 

```ts

// bestNumber - Current Block
// end - Block at which crowdload ends

// Show remaining days only if end is after the current block.

const blocksLeft = useMemo(
    () => bestNumber && end.gt(bestNumber)
      ? end.sub(bestNumber)
      : null,
    [bestNumber, end]
  );
```
